lib/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
   "compilerOptions": {
+    "baseUrl": "..",
```

Comment:
does this mean we will need to change the `outDir`/source or is that still fine?